src/main/java/seedu/address/model/AppointmentSchedule.java

```diff
@@ -67,6 +67,15 @@ public boolean hasAppointment(Appointment appointment) {
         return appointments.contains(appointment);
     }
 
+    /**
+     * Returns true if an appointment with the same identity as {@code appointment} exists
+     * in the appointment schedule.
+     */
+    public boolean hasEditAppointment(Appointment appointment) {
```

Comment:
I don't think this is needed since it functions the same as `hasAppointment` method.

Reply:
i think they are different because the editContains is different from Contains

Reply:
you are right. Removed duplicate methods